src/js/actions/search/menucommands.js

```diff
@@ -53,7 +53,16 @@ define(function (require, exports) {
             }
 
             if (labels[part].$MENU) {
-                path += labels[part].$MENU + ">";
+                // Replace "Application" with "Photoshop CC" on Macs
+                // Don't show "Application" options on Windows
+                if (part !== "APPLICATION") {
+                    path += labels[part].$MENU + ">";
+                } else if (system.isMac) {
+                    path += "Photoshop CC >";
```

Comment:
At the very least this needs to be localized. But, what happens if you just change this line? https://github.com/adobe-photoshop/spaces-design/blob/master/src/nls/root/strings.js#L68